table.lua

```diff
@@ -501,7 +533,7 @@ function flib_table.slice(arr, start, stop)
 
   local k = 1
   for i = start, stop do
-    output[k] = arr[i]
+    output[k] = array[i]
     k = k + 1
   end
   return output
```

Comment:
Micro perfomance change, Eval after fail state, Output after fail state. Previously if it failed there would be 2 tables created(output and {})